src/keylessBackup/KeylessBackupProgress.tsx

```diff
@@ -341,7 +342,11 @@ function Setup({ origin }: { origin: KeylessBackupOrigin }) {
 
   const onPressSkip = () => {
     AppAnalytics.track(KeylessBackupEvents.cab_progress_failed_skip_onboarding)
-    navigate(Screens.ChooseYourAdventure)
+    if (CYA_ENABLED) {
+      navigate(Screens.ChooseYourAdventure)
+    } else {
+      navigateHome()
+    }
```

Comment:
can this use the onboarding/steps for getting the next step? We want all onboarding to end in the success screen, and only the success screen should navigate to CYA. This was missed in the fix I made earlier.